e2e-tests/insurance/cypress/e2e/journeys/policy/broker-addresses/broker-addresses-page.spec.js

```diff
@@ -69,12 +76,47 @@ context('Insurance - Policy - Broker addresses page', () => {
       });
 
       it('renders an `address` radio label and input', () => {
-        const expectedFieldValue = 'BRITISH BROADCASTING CORPORATION WOGAN HOUSE PORTLAND PLACE';
-
         const { option } = radios(FIELD_ID, expectedFieldValue);
 
         cy.checkText(option.label(), expectedFieldValue);
       });
     });
   });
+
+  describe('form submission', () => {
+    describe('when submitting an empty form', () => {
+      const errorIndex = 0;
+      const expectedErrorMessage = ERROR_MESSAGES.INSURANCE.POLICY[FIELD_ID].IS_EMPTY;
+
+      beforeEach(() => {
+        cy.navigateToUrl(url);
+        cy.clickSubmitButton();
+      });
+
+      it('should render validation errors', () => {
+        const field = fieldSelector(FIELD_ID);
+
+        cy.checkErrorSummaryListHeading();
+        cy.assertErrorSummaryListLength(1);
```

Comment:
did we not have a shared command for checking these errors?

Reply:
We have a sparsely used`cy.assertFieldErrors`, but it will not work for radios.

Reply:
submitAndAssertRadioErrors?

Reply:
Tried - that won't work for this - it relies on a single field ID - the field is called `selectTheAddress` but none of the options are called this because they are dynamic addresses.

We have this in "Type of policy" as well for example which have "single" and "multiple" radio options.